infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -1,47 +1,63 @@
 package org.corfudb.infrastructure;
 
-import com.github.benmanes.caffeine.cache.Cache;
-import com.github.benmanes.caffeine.cache.CacheWriter;
-import com.github.benmanes.caffeine.cache.Caffeine;
-import com.github.benmanes.caffeine.cache.RemovalCause;
-import com.google.common.annotations.VisibleForTesting;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
+import javax.annotation.concurrent.NotThreadSafe;
 import org.corfudb.runtime.view.Address;
+import org.corfudb.util.MetricsUtils;
 
-import javax.annotation.Nonnull;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.PriorityQueue;
 import java.util.UUID;
-import java.util.concurrent.atomic.AtomicLong;
 
 /**
  * Sequencer server cache.
  * Contains transaction conflict-resolution data structures.
- * <p>
  * The SequencerServer use its own thread/s. To guarantee correct tx conflict-resolution,
- * the {@link SequencerServerCache#conflictCache} must be updated
+ * the {@link SequencerServerCache#cacheConflictKeys} must be updated
```

Comment:
Halve of the JavaDoc is not relevant anymore (because you replaced caffeine by a hashmap)

Reply:
Clean up the information